float32 y_axis_motion
---
bool spray_done
